docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/BinaryClassification/FastTree.cs

```diff
@@ -87,13 +100,18 @@ private static IEnumerable<DataPoint> GenerateRandomDataPoints(int count, int se
                 {
                     Label = label,
                     // Create random features that are correlated with the label.
-                    // For data points with false label, the feature values are slightly increased by adding a constant.
-                    Features = Enumerable.Repeat(label, 50).Select(x => x ? randomFloat() : randomFloat() + 0.03f).ToArray()
+                    // For data points with false label, the feature values are
+                    // slightly increased by adding a constant.
+                    Features = Enumerable.Repeat(label, 50)
+                        .Select(x => x ? randomFloat() : randomFloat() +
+                        0.03f).ToArray()
+            
```

Comment:
Will that entire ternary operator fit on one line? I think (x => x ? randomFloat() : randomFloat() + 0.03f) should be on one line, if it fits.